mlir/include/mlir/Interfaces/TilingInterface.td

```diff
@@ -76,11 +112,32 @@ def TilingInterface : OpInterface<"TilingInterface"> {
       >,
       InterfaceMethod<
         /*desc=*/[{
-          Method to return the position of the result tile computed by the tiled operation.
+          Method to return the position of the result tile computed by the
+          tiled operation.
+
+          For operations that return a value (typically a value of type
+          `RankedTensorType`), the generated tiled computation has to also
+          recompute a replacement for the results of the original operation.
+          The tiled implementation of the operation returns a tile of the
+          result(s). This methods returns information about what part of the
+          result tensor is computed by the tiled implementation. The manner in
+          which these tiles get put together to get the final result is upto
+          the surrounding loop construct.
+          - `resultNumber` is the result number of the original operation
```

Comment:
`result number` seems like non-standard notation to me. Maybe `index of the OpResult`?

Reply:
`OpResult` has a method called `resultNumber`... so pretty standard?

Reply:
Oh I see, I had thought it was `index`. `resultNumber` is fine then.